### ISSUE TITLE
transport_ofi.c:336:  ep_bind mr2epfd failed [001] ERROR: Transport init failed

### ISSUE THREAD

**Issue by jeffhammond** [Sandia-OpenSHMEM/SOS#210]
PRK is broken by some recent commit to SOS-OFI...

From https://travis-ci.org/ParRes/Kernels/jobs/140153878:
```sh
cd SHMEM/Transpose;         make transpose "DEFAULT_OPT_FLAGS   = -O3"
make[1]: Entering directory `/home/travis/build/ParRes/Kernels/SHMEM/Transpose'
/home/travis/build/ParRes/Kernels/PRK-deps/sandia-openshmem/bin/oshcc -O3 -DSHMEM    -I../../include -c transpose.c
/home/travis/build/ParRes/Kernels/PRK-deps/sandia-openshmem/bin/oshcc -O3 -DSHMEM    -I../../include -c ../../common/wtime.c
/home/travis/build/ParRes/Kernels/PRK-deps/sandia-openshmem/bin/oshcc -O3 -DSHMEM    -I../../include -c ../../common/SHMEM_bail_out.c
/home/travis/build/ParRes/Kernels/PRK-deps/sandia-openshmem/bin/oshcc -o transpose   -O3 -DSHMEM transpose.o wtime.o SHMEM_bail_out.o  -lm
make[1]: Leaving directory `/home/travis/build/ParRes/Kernels/SHMEM/Transpose'
+export PRK_TARGET_PATH=SHMEM
+export PRK_SHMEM_PROCS=4
+export OSHRUN_LAUNCHER=/home/travis/build/ParRes/Kernels/PRK-deps/hydra/bin/mpirun
+export PRK_LAUNCHER=/home/travis/build/ParRes/Kernels/PRK-deps/sandia-openshmem/bin/oshrun
+/home/travis/build/ParRes/Kernels/PRK-deps/sandia-openshmem/bin/oshrun -n 4 SHMEM/Synch_p2p/p2p 10 1024 1024
transport_ofi.c:336: 
ep_bind mr2epfd failed
[002] ERROR: Transport init failed
transport_ofi.c:336: 
ep_bind mr2epfd failed
[003] ERROR: Transport init failed
transport_ofi.c:336: 
ep_bind mr2epfd failed
[001] ERROR: Transport init failed
transport_ofi.c:336: 
ep_bind mr2epfd failed
[000] ERROR: Transport init failed
===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   PID 32661 RUNNING AT testing-worker-linux-docker-9844e460-3372-linux-16
=   EXIT CODE: 134
=   CLEANING UP REMAINING PROCESSES
=   YOU CAN IGNORE THE BELOW CLEANUP MESSAGES
===================================================================================
YOUR APPLICATION TERMINATED WITH THE EXIT STRING: Aborted (signal 6)
This typically refers to a problem with your application.
Please see the FAQ page for debugging suggestions
```

**Comment by jeffhammond** [comment]
We are using v1.2.0 so this might be an issue with OFI.  Please hold while I investigate further.

**Comment by kseager** [comment]
I can look into it; It does look like an OFI-sockets regression

**Comment by jeffhammond** [comment]
The bug went away when I dropped back to libfabric 1.3.0.

**Comment by kseager** [comment]
fixed with PR #212
